Library/Formula/refind.rb

```diff
@@ -0,0 +1,9 @@
+class Refind < Formula
+  homepage "http://www.rodsbooks.com/refind/"
+  url "https://downloads.sourceforge.net/project/refind/0.8.4/refind-bin-0.8.4.zip"
+  sha1 "f219383924ef60cb6d147267f9ee44810c1f473f"
+
+  def install
+    system "sudo", "bash", "install.sh"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!